ENIGMAsystem/SHELL/Bridges/General/DX9Context.h

```diff
@@ -249,13 +247,17 @@ void Release() {
 
 void Reset(D3DPRESENT_PARAMETERS *pPresentationParameters) {
 	HRESULT hr = device->Reset(pPresentationParameters);
-		if(FAILED(hr)){
-			MessageBox(enigma::hWnd,
-               "Failed to reset Direct3D 9.0 Device",
-			   DXGetErrorDescription9(hr), //DXGetErrorString9(hr)
-               MB_ICONERROR | MB_OK);
-			return;  // should probably force the game closed
-		}
+	if (FAILED(hr)) {
+		MessageBox(
```

Comment:
How is this different from `show_error`?

Reply:
We can change it to `show_error` another time, this entire file is going to be deleted and moved. I'll do it at that time that I go to salvage stuff in here.